treeherder/model/management/commands/cycle_data.py

```diff
@@ -81,27 +82,17 @@ class PerfherderCycler(DataCycler):
     def __init__(self, days, chunk_size, sleep_time, is_debug=None,
                  logger=None, **kwargs):
         super().__init__(days, chunk_size, sleep_time, is_debug, logger)
-        self.started_at = None
+        if days < MINIMUM_PERFHERDER_EXPIRE_INTERVAL:
+            raise ValueError('Cannot expiry too recent performance data')
```

Comment:
Perhaps it'd be useful to show what the limit is by including the variable: "Cannot expire data earlier than <expire interval> days"?

Reply:
Yes, that'd be useful to know from the logs.